packages/unified-selection/README.md

```diff
@@ -166,3 +166,21 @@ const queryExecutor = createECSqlQueryExecutor(imodel);
 // Returns the parent element, or the element itself if it does not have a parent, for each element specified in `elementIds` argument.
 const selection = computeSelection({ queryExecutor, elementIds, scope: { id: "element", ancestorLevel: 1 } });
 ```
+
+## Viewport synchronization with unified selection
```

Comment:
I'm not sure it's correct to say that we're syncing **viewport** with unified selection, when we're actually syncing **iModel's selection set** with unified selection. 

Also, I believe saying that this hook should be used in a viewport component is not correct. What's going to happen if there are two or more viewports for the same iModel?

Reply:
We are also syncing iModel's hilited set, so I was not sure what a good title for it would be. Should the `syncViewportWithUnifiedSelection` function be renamed as well?

Regarding the multiple viewports, in the previous implementation `SelectionManager` had the methods `setSyncWithIModelToolSelection` and `suspendIModelToolSelectionSync` which were responsible for registering the listeners and ensuring that only one was active at a time. I did not add an equivalent for it as I am not sure what is the use case for attempting to synchronize from multiple places at the same time. In the current implementation it would be problematic if the function is called more than once, regardless whether it is in a viewport component or not. Would it be better to then mention that it should only be called once?

Reply:
> We are also syncing iModel's hilited set, so I was not sure what a good title for it would be. Should the `syncViewportWithUnifiedSelection` function be renamed as well?

Yes, the function should not lie about what it does. Also, what I don't like about `syncViewportWithUnifiedSelection` is that it sounds like a one time action, when in fact it's enabling a continuous sync. I think the name could be, for example `enableUnifiedSelectionSyncWithIModel`. It's not great, so I welcome suggestions for better names.

> In the current implementation it would be problematic if the function is called more than once, regardless whether it is in a viewport component or not. Would it be better to then mention that it should only be called once?

I think the function being clear that it's syncing with an IModel rather than a Viewport is a good first step. The next step is API docs and README making it clear what it does and providing a good example.

Reply:
I think `enableUnifiedSelectionSyncWithIModel` is good enough. Renamed the functions to better reflect what they are doing. Updated docs and README.